<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addScientist</class>
 <widget class="QDialog" name="addScientist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>272</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string>Add person</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameText">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string>Name: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameInput">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="genderText">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string>Gender: </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="maleButton">
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="text">
        <string>Male</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="femaleButton">
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="yobText">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string>Year of birth:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="yobInput">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="yodText">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string>Deceased: </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="deadButton">
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="aliveButton">
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="text">
        <string>No</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="yodText_2">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string>Year of death:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="yodInput">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameInput</tabstop>
  <tabstop>maleButton</tabstop>
  <tabstop>femaleButton</tabstop>
  <tabstop>yobInput</tabstop>
  <tabstop>deadButton</tabstop>
  <tabstop>aliveButton</tabstop>
  <tabstop>yodInput</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
